src/getMongoLocker.ts

```diff
@@ -30,7 +39,12 @@ export default async (collection: Collection) => {
 
         const acquireLock = async () => {
             try {
-                await collection.insertOne({ _id: key, lockId, acquiredAt: new Date() });
+                const now = new Date();
+                await collection.replaceOne(
```

Comment:
Please describe the logic of this query with comment - how exactly it identifies the lock and how it will throw an error.